presto-main/src/main/java/com/facebook/presto/metadata/SqlScalarFunctionBuilder.java

```diff
@@ -198,6 +198,11 @@ public Long getLiteral(String name)
             return boundVariables.getLongVariable(name);
         }
 
+        public boolean containsLiteral(String name)
```

Comment:
I am not sure this is needed. I think `VARCHAR` should always have bounded length. Pure `VARCHAR` has length `Integer.MAX_VALUE`